tractor/__init__.py

```diff
@@ -99,15 +100,18 @@ def run(
     name: Optional[str] = None,
     arbiter_addr: Tuple[str, int] = (
         _default_arbiter_host, _default_arbiter_port),
-    # the `multiprocessing` start method:
+    # either the `multiprocessing` start method:
     # https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods
-    start_method: str = 'forkserver',
+    # OR `trio-run-in-process` (the new default).
+    start_method: str = 'trip',
```

Comment:
This comment is confusing, since it seems like its saying that `trio-run-in-process` is the new default, but it's actually the shortened version `trip` that is the default. I'd pick one or the other as the "correct" spelling in the code.

Reply:
Do you think the long or short name is better?

Reply:
I like the short name better, but I think it would be best to be consistent with the package name. Maybe we'll eventually get a package with a shorter, more user friendly name. Then I'll be very happy to see us using that name instead.

Reply:
@ryanhiebert went with the explicit name as asked.